### DISCUSSION TITLE
Reset layout

### DISCUSSION THREAD

**Discussion by Midhil457** [4ian/GDevelop#3165] [Enhancement ideas (and their technical discussions)]
## Description

It nessesary to have a reset layout option to reset the mosaic in the scene editor. It annoyingly not possible to re-create the default layout (The layer window specifically). Yes, after some tries, it's possible to re-make it. But, it's annoying to the user, annoyence isn't what everyone wants especially when developing a game (debuging, testing, late night work, rage quits, etc..) I experimented with changing up the layout and now I can't go back to the default (Which I prefered)

## Solution suggested
Have a action in the `View` section in the toolbar

**Comment by Bouh** [comment]
This kind of reset button in preferences?
![image](https://user-images.githubusercontent.com/1670670/136665766-c4805716-0609-4711-8cb7-5abf0f11f9c3.png)

**Comment by Midhil457** [comment]
😅 I didn't remember that. But, I guess it will be more obvious to also have it under 'View' in the toolbar as most people would be looking there for anything related to the UI (View)
